pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -26,15 +26,16 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/jpillora/backoff"
-	"github.com/kanisterio/kanister/pkg/poll"
 	"github.com/pkg/errors"
 
```

Comment:
[nit] fix imports

Reply:
I assumed we wanted to group all the kanister imports together so I moved it to the list below?

Reply:
Oh sorry. I misread it. The imports are fine